README.md

```diff
@@ -312,7 +312,7 @@ desired points in the life of a virtual node.
 The following hooks are available for modules: `pre`, `create`,
 `update`, `destroy`, `remove`, `post`.
 
-The following hooks are available in the `hook` property of individual
+The following hooks are available in the `hook` property of the individual
```

Comment:
I don't think the "the" is appropriate here as the sentence is not referring to specific elements but just all elements in general?